NetworkPkg/UefiPxeBcDxe/PxeBcMtftp.c

```diff
@@ -48,7 +48,13 @@ PxeBcMtftp6CheckPacket (
   EFI_PXE_BASE_CODE_CALLBACK_PROTOCOL  *Callback;
   EFI_STATUS                           Status;
 
-  Private  = (PXEBC_PRIVATE_DATA *)Token->Context;
+  Private = (PXEBC_PRIVATE_DATA *)Token->Context;

```

Comment:
Looks like a space got removed here, if this wasn't done by Uncrustify, can we leave it as is? No need to change the blame for it.